modules/ROOT/pages/pricing.adoc

```diff
@@ -0,0 +1,80 @@
+
+== Metrics Definitions
+
+Anypoint Platform has a pricing package that is based on consumption of allotted Mule components.
+
+For information about pricing concepts within Anypoint Platform, see xref:general::pricing-metrics.adoc[Pricing and Usage Metrics].
+
+// WIP
+
+== Global Resource Pool Limits
+
+Root organizations provisioned with the new pricing and packaging receive a global pool of shared CPU and memory resources that is shared for deployments into CloudHub 1.0 and CloudHub 2.0 across all business groups and environments in the organization. The pool of resources is sufficiently large to consume all the purchased flows and messages without interruption. If an organization runs out of resources, actions that consume more resources such as deploying new applications or starting stopped applications are blocked until sufficient resources are available. 
+
+You can free up resources by stopping or deleting unused applications, or reducing the replica size or resources that an existing application consumes. You receive a warning each time your organization approaches the resource limit on the Applications List UI, Application Deployment UI, and Manage Application UI. To avoid hitting limits, allocate only what you need by using granular replica sizes in CloudHub 1.0 and CloudHub 2.0. Contact Support if you require an exception when you reach the limit. 
+
+== CloudHub 2.0 Application Deployment
+
+CloudHub 2.0 now uses replica sizing for application deployment. You can choose from the following replica sizes based on the application profile or size and display the equivalent in terms of vCores. Replica sizes with the suffix `.mem` are memory-intensive options with the same compute. 
+
+include::partial$pricing.adoc[tag=pricingTable]
+
+During the monthly patching cycle, your applications automatically migrate to an equivalent replica size. Application deployment through the Mule Maven Plugin is not yet supported. However, you can deploy using the Runtime Manager UI and the Application Manager Experience APIs by leveraging a new field under `target.deploymentSettings` called `instanceType`.
+
+== Access Management Entitlements
+
+Accounts provisioned with the new pricing and packaging show flows, messages, and throughput entitlements on Access Management instead of vCores. These entitlements are available at the root organization level and cannot be allocated among business groups. Because vCores are no longer the purchased entity, they can no longer be tracked or allocated among business groups. To view your monthly usage of flows, messages, and throughput, xref:general::usage-reports.adoc#access-usage-reports[access usage reports] using the Access Management UI. 
```

Comment:
```suggestion
Accounts provisioned with the new pricing and packaging show flows, messages, and throughput entitlements instead of vCores on Access Management. These entitlements are available at the root organization level and cannot be allocated among business groups. Because vCores are no longer the purchased entity, they can no longer be tracked or allocated among business groups. To view your monthly usage of flows, messages, and throughput, xref:general::usage-reports.adoc#access-usage-reports[access usage reports] by using the Access Management UI. 
```